tests/helpers/util.py

```diff
@@ -0,0 +1,350 @@
+import asyncio
+import io
+import os
+import random
+import re
+import socket
+import subprocess
+import tarfile
+import threading
+import time
+from contextlib import contextmanager
+from functools import partial as p
+from io import BytesIO
+from typing import Dict, List
+
+import docker
+import netifaces as ni
+from tests.helpers.assertions import regex_search_matches_output
+from tests.paths import TEST_SERVICES_DIR
+
+DEFAULT_TIMEOUT = int(os.environ.get("DEFAULT_TIMEOUT", 30))
+DOCKER_API_VERSION = "1.34"
+STATSD_RE = re.compile(r"SignalFx StatsD monitor: Listening on host & port udp:\[::\]:([0-9]*)")
```

Comment:
THis is definitely not needed.  Cull this whole file to only the absolute necessarye